pkg/lockfile/parse-gradle-lock.go

```diff
@@ -27,13 +27,14 @@ func parseToGradlePackageDetail(line string) (PackageDetails, error) {
 		return PackageDetails{}, fmt.Errorf("invalid line in gradle lockfile: %s", line)
 	}
 
-	group, artifact, version := parts[0], parts[1], parts[2]
-	version = strings.SplitN(version, "=", 2)[0]
+	group, artifact, versionParts := parts[0], parts[1], strings.SplitN(parts[2], "=", 2)
+	version, scopes := versionParts[0], strings.Split(versionParts[1], ",")
 
```

Comment:
I think it'd be clearer to use `Cut` instead of `SplitN`, even though the return types are different, the goal here is to split between the version and the comma-separated scopes - we're only splitting a string by the first occurrence of the equal sign, so to me it'd be a lot clearer if we use `Cut`, and then split the right-hand side of the string by commas.